src/pages/_app.tsx

```diff
@@ -14,12 +12,11 @@ export default function App({
   pageProps: { session, ...pageProps },
 }: AppProps) {
   return (
-    <ChakraProvider theme={chakraTheme}>
```

Comment:
Can we add ChakraProvider back in? I think the game screen breaks without it

Reply:
Oh oops - didn't know when that was added in. My understanding was that the component that wanted to use Chakra would wrap itself in a ChakraProvider. Otherwise you get some weird behavior with default themes

Reply:
You can try it with this branch - if you re-add this code it overrides the existing styling (global fonts, etc)

Reply:
Feel free!